usage-api/nodejs/index.js

```diff
@@ -0,0 +1,62 @@
+// Get the needed environment variables
+require('dotenv').config();
+let apiKey = process.env.APIKEY;
+let contentType = 'application/json'
+let appId = process.env.APPID;
+let region = process.env.REGION;
+let orgId = process.env.ORGID;
+
+// Function calls to the three usage api
+overallCustomerCount();
+usagePerCustomerWithinRegion();
+individualCustomerSubscription();
+
+let request = require('request');
+
+// Function to get the overall customer count 
+function overallCustomerCount() {
+    let options = {
+        'method': 'GET',
+        'url': 'https://billable-vendor-usage-api.usw2.pure.cloud/v1/regions?appIds=' + appId,
+        'headers': {
+            'Content-Type': contentType,
+            'x-api-key': apiKey
+        }
+    };
+    request(options, function (error, response) { 
+        if (error) throw new Error(error);
+        console.log(response.body);
```

Comment:
It would be nice to add some context to the log message to indicate which response is being traced out. Because the API requests are asynchronous, the responses logged to the console may not be in the same order in which they were requested.

Reply:
That is absolutely correct. i have updated it now